ItalianTranslation_PlanetCoaster.xml

```diff
@@ -14123,7 +14123,7 @@
 		<entry key="ParkManagement_ExpenditureCategory_Uncategorised" translation="Non categorizzato"/>
 		<entry key="ParkManagement_Facilities" translation="Strutture"/>
 		<entry key="ParkManagement_FamilyGroups" translation="Gruppi Famiglia"/>
-		<entry key="ParkManagement_Finance" translation="Finanze"/>
+		<entry key="ParkManagement_Finance" translation="Economia"/>
```

Comment:
Sicuro che "Economia" ci stia meglio(?) Bah sì in effetti sono intercambiabili

Reply:
Ho visto che in altri punti era tradotto così..